### DISCUSSION TITLE
How can I watch the paths' addresses for further analysis in a disassembler

### DISCUSSION THREAD

**Discussion by MaorRakach** [AFLplusplus/AFLplusplus#1123] [Q&A]
I'm fuzzing a huge framework composed of multiple shared libraries (.so files). I would like to view the paths AFL++ finds in the form of something like libMyLib.so:0x4000ef so I can cross check it with a disassembler and view the results; Is there way to pull out that data?

**Comment by vanhauser-thc** [comment]
This is a binary-only fuzzing project? if so, qemu or frida? or can you use either?
